netbeans-plugin/nb-configuration.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
License header added. This file generate by netbeans. Project doesn't build without this file.